test/e2e/Apps/BasicDotNetIsolated/OrchestrationQuery.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Net;
+using Grpc.Core;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using Microsoft.DurableTask.Client;
+
+namespace Microsoft.Azure.Durable.Tests.E2E;
+public static class OrchestrationQueryFunctions
+{
+    [Function(nameof(GetAllStatus))]
+    public static async Task<HttpResponseData> GetAllStatus(
```

Comment:
nit: Consider naming this `GetAllInstances`

Reply:
Done